### PR TITLE
[SPARK-11188][SQL] Elide stacktraces in bin/spark-sql for AnalysisExceptions

### PR DESCRIPTION
**Pull request by dilipbiswal**
Only print the error message to the console for Analysis Exceptions in sql-shell

### CODE REVIEW

**Comment by dilipbiswal** [comment]
@marmbrus this is for 1.5.2 branch.

**Comment by marmbrus** [comment]
ok to test

**Comment by marmbrus** [comment]
Thanks, merging

**Comment by marmbrus** [comment]
Can you close this.  PRs not against master don't auto close.

File: sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala [hunk 1]
```diff
@@ -22,6 +22,8 @@ import scala.collection.JavaConversions._
 import java.io._
 import java.util.{ArrayList => JArrayList, Locale}
 
+import org.apache.spark.sql.AnalysisException
```
Comment (marmbrus) [inline sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala]: ordering

File: sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala [hunk 2]
```diff
@@ -308,7 +311,12 @@ private[hive] class SparkSQLCLIDriver extends CliDriver with Logging {
 
           ret = rc.getResponseCode
           if (ret != 0) {
-            console.printError(rc.getErrorMessage())
+            // For analysis exception, only the error is printed out to the console.
+            rc.getException() match {
+              case e : AnalysisException =>
```
Comment (marmbrus) [inline sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala]: no space after `e`

**Comment by dilipbiswal** [comment]
@marmbrus Not able to figure out what the problem is here. Can you please advise ? I think it also needs a retest.

**Comment by marmbrus** [comment]
Sorry, I merged this one before noticing the small style issues, so you'll need to do a new PR if you want to fix them.  Eitherway, can you click `Close` on this PR?

**Comment by dilipbiswal** [comment]
@marmbrus Michael, surely i would like fix the style issues. I am closing this one. Can you please let me know what the problems are ? I would fix them right away. Thanks a lot.

**Comment by marmbrus** [comment]
You fixed them here: https://github.com/dilipbiswal/spark/commit/4783665a94b092490aa88156ef9d40b8327943ed

I had already merged the patch though.

**Comment by dilipbiswal** [comment]
@marmbrus Michael, should i open a minor JIRA to fix up the scala style issue for 1.5.2 ?

**Comment by marmbrus** [comment]
Yeah, its good to track everything that changes code incase it gets reverted.

**Pull request closed without merging**